plugins/generic/datacite/DataciteExportPlugin.php

```diff
@@ -242,6 +242,12 @@ public function exportAsDownload(Context $context, array $objects, ?bool $noVali
      */
     public function depositXML($object, $context, $filename)
     {
+        // Application is set to sandbox mode and will not run the features of plugin
+        if (Config::getVar('general', 'sandbox', false)) {
+            error_log('Application is set to sandbox mode and datacite will not do any deposition');
+            return [[__('common.sandbox')]];
```

Comment:
Have you tested this? I think this will result in double-translation of the text, looking at the other return values below.